### PR TITLE
add 2024 VC API Implementation annual review

### PR DESCRIPTION
**Pull request by jrhender**
This PR adds the 2024 annual review for the VC API Implementation project.

Project repository: https://github.com/openwallet-foundation-labs/vc-api/

### CODE REVIEW

File: docs/projects/reviews/2024/2024-vc-api-implementation-annual.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+GitHub Issue: https://github.com/openwallet-foundation-labs/vc-api/issues/29 
+
+This item aims to improve the ability of technology selection decision makers, such as CTOs or Software Architects, to assess whether or not the VC API implementation is suitable for their use cases and technology context.
+The key output is a new documentation site.
+
+#### Listing in the VC Playground
+
+GitHub Issue: https://github.com/openwallet-foundation-labs/vc-api/issues/28
+
+The VC Playground is a demonstration application for Verifiable Credential issuance and verification: https://vcplayground.org.
```
Comment (tkuhrt) [inline docs/projects/reviews/2024/2024-vc-api-implementation-annual.md:68]: Who supports/runs the VC Playground?

**Review by tkuhrt — APPROVED** [review]
Thanks for the annual review of VC-API. I will add this to the next TAC meeting for discussion purposes. I did have one question that I am hoping that you can answer.

File: docs/projects/reviews/2024/2024-vc-api-implementation-annual.md (hunk 1 above)
Comment (jrhender) [reply docs/projects/reviews/2024/2024-vc-api-implementation-annual.md:68]: @tkuhrt I know that Digital Bazaar assists with running and developing the VC Playground. There may be other supporters that I am not aware of.

File: docs/projects/reviews/2024/2024-vc-api-implementation-annual.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+The underlying [VC API specification](https://w3c-ccg.github.io/vc-api/) (currently developed in the context of the W3C Credentials Community Group) is active and the specification group has the intention of bringing the specification to a W3C standards track in the months ahead.
+
+## Maintainer Diversity
+
+The project currently has four maintainers, from two organizations, Energy Web & Impactility.
+
+## Project Adoption
+
+The project currently one primary adoption organization, Elia Group. Elia continues to showcase the VC API project on their SSI demonstration site (https://ssi4energy.eliagroup.eu/en).
```
Comment (skounis) [inline docs/projects/reviews/2024/2024-vc-api-implementation-annual.md:19]: It looks like the verb is missing "The project currently **has** one primary adoption organization, Elia Group"

Comment (jrhender) [reply docs/projects/reviews/2024/2024-vc-api-implementation-annual.md]: Thanks @skounis , fixed pushed here: [8078832](https://github.com/openwallet-foundation/tac/pull/185/commits/8078832996a417219641d4ef8f331ef354b2c9b4)

**Review by skounis — APPROVED** [review]

**Pull request merged**